src/Chapter1.hs

```diff
@@ -619,8 +635,17 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
+sumLast2 :: Int -> Int
+sumLast2 n = secondToTheLastDigit n + lastDigit n
+  where
+    lastDigit :: Int -> Int
+    lastDigit a = mod a 10
 
-sumLast2 n = error "sumLast2: Not implemented!"
+    secondToTheLastDigit :: Int -> Int
+    secondToTheLastDigit b
+        | intermediateNumber > 9 =  mod intermediateNumber 10
+        | otherwise = intermediateNumber
+        where intermediateNumber = div b 10
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: you can see that you use both:

```haskell
mod (abs n) 10
div (abs n) 10
```
The standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(first, second) = divMod (abs n) 10
```

You can see how we could pattern match on the pair 🙂